rsp_jupyter_extensions/handlers/ghostwriter.py

```diff
@@ -28,12 +31,20 @@ def prepare(self) -> None:  # type: ignore[override]
         """Issue a redirect based on the request path."""
         # the implicit None return can also function as a null coroutine,
         # and in Python 3.13, "None" becomes a valid return type from it.
+        #
+        # So once we're at Python 3.13, we can remove that type: ignore.
         redir = _peel_route(self.request.path, "/rubin/ghostwriter")
+        ext_url = os.getenv("EXTERNAL_INSTANCE_URL", "http://localhost:8888")
```

Comment:
The fallback is never going to work, is it? Wouldn't it be better to raise an exception here? Who knows what the user may have running on localhost.

Reply:
What the user _probably_ has running is Jupyter Lab, locally, because that's where it runs as default.  However, yeah, if the user is running locally, then they don't have a working ghostwriter service, so, you're right, it should raise an exception.